resources/views/dashboard/index.blade.php

```diff
@@ -79,22 +34,55 @@
         </div>
       @endif
 
-      <div class="row row-compressed">
-        @if (FixometerHelper::hasRole($user, 'Administrator'))
-          @include('dashboard.restarter')
-        @endif
-        @if (FixometerHelper::hasRole($user, 'Host'))
-          @include('dashboard.host')
-        @endif
-        @if (FixometerHelper::hasRole($user, 'Restarter'))
-          @include('dashboard.restarter')
-        @endif
-        @if (FixometerHelper::hasRole($user, 'NetworkCoordinator'))
-            @include('dashboard.coordinator')
-        @endif
-        <div class="col-12">
-            @include('dashboard.blocks.impact')
-        </div>
+      <?php
+        function expandGroups($groups) {
```

Comment:
I have a memory that we said we'd try to move this type of expansion logic in to the controller, and pass the expanded result in to the view, going forwards - should we?  

That said, in this case, where is expandGroups actually called?

Reply:
I think I left this code in by mistake having moved the logic into the controller.  Removed it now.